{"name": "test_case[case3]", "status": "broken", "statusDetails": {"message": "AttributeError: 'SubRequest' object has no attribute 'param'", "trace": "request = <SubRequest 'init_api' for <Function test_case[case3]>>\n\n    @pytest.fixture(scope=\"function\")\n    # TODO fixture 一般是放在conftest中，只能是conftest；在外面 每个测试用例执行都能用执行时会自动发现conftest的fixture，\n    # 不用在测试用例中导入conftest\n    def init_api(request):\n        '''获得初始值与测试数据'''\n        do_exl = DoExcel(project_path.case_path,\"Login\")\n        my_log = MyLog()\n>       return request.param,do_exl,my_log   # 固定，每调用一次fixture，就取出一个数据,返回的是一个元祖\nE       AttributeError: 'SubRequest' object has no attribute 'param'\n\ntest_run_02_login_new2.py:34: AttributeError"}, "description": "case 是上面传的测试用例数据", "parameters": [{"name": "case", "value": "{'CaseId': 4, 'Module': 'Login', 'Title': '输入错误的手机号', 'Url': 'http://test.lemonban.com/futureloan/mvc/api/member/login', 'Method': 'post', 'Params': \"{'mobilephone':'186887756569','pwd':'123456'}\", 'ExpectedResult': \"{'status':0,'code':'20111','data': None,'msg':'用户名或密码错误'}\"}"}], "start": 1573479663506, "stop": 1573479663506, "uuid": "5bdf0c0f-deb0-453c-8df8-8fc922a34e31", "historyId": "adb14627a80a67ed86403d921973a1f4", "testCaseId": "f1fc86fa6c22953baf4b8542c7a8aa2d", "fullName": "test_run_02_login_new2.TestCase#test_case", "labels": [{"name": "tag", "value": "moke"}, {"name": "suite", "value": "test_run_02_login_new2"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "LAPTOP-D1LKV60G"}, {"name": "thread", "value": "4476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_run_02_login_new2"}]}